src/components/display/fewlines/NeutralLink.tsx

```diff
@@ -5,6 +5,7 @@ import style from "styled-components";
 interface ExtendedLinkProps extends LinkProps {
```

Comment:
didn't it already exists on `LinksProps`? Seems like it should since it would be a property of anchor tags.

Reply:
Nope, not on `LinkProps`. It exists on the type of `<a>` tags, but since we need to pass it to the custom component `NeutralLink` (Next's `Link` > usual `<a>` tag ) in its props, I needed to declare it in its extended props (as for `onClick` method for example)